src/main/java/net/minecraftforge/client/model/MultiLayerModel.java

```diff
@@ -269,13 +274,11 @@ public MultiLayerModel read(JsonDeserializationContext deserializationContext, J
         {
             ImmutableList.Builder<Pair<RenderType, UnbakedModel>> builder = ImmutableList.builder();
             JsonObject layersObject = GsonHelper.getAsJsonObject(modelContents, "layers");
-            for(Map.Entry<String, RenderType> layer : BLOCK_LAYERS.entrySet()) // block layers
-            {
-                String layerName = layer.getKey(); // mc overrides toString to return the ID for the layer
-                if(layersObject.has(layerName))
-                {
-                    builder.add(Pair.of(layer.getValue(), deserializationContext.deserialize(GsonHelper.getAsJsonObject(layersObject, layerName), BlockModel.class)));
-                }
+            var renderTypes = LevelRenderPhaseManager.getInstance().getAllKnownTypes();
+            for (var key : layersObject.keySet()) {
```

Comment:
Formatting: braces on newlines:
```suggestion
            for (var key : layersObject.keySet())
            {
```

Reply:
This is little off topic but why doesnt this repo have an .editorconfig file?

Reply:
Because style is a weird and wacky place - patches and code have different styles, and no matter what we provide some people will choose to override it with their own preference anyway, which makes the whole thing moot.